src/isomorphic/classic/types/__tests__/ReactPropTypesProduction-test.js

```diff
@@ -23,7 +23,8 @@ describe('ReactPropTypesProduction', function() {
     __DEV__ = false;
     oldProcess = process;
     global.process = {
-      env: Object.assign({}, process.env, {NODE_ENV: 'production'}),
+      cwd: process.cwd,
+      env: Object.assign({}, process.env, {NODE_ENV: JSON.stringify('production')}),
```

Comment:
I don't think you need `JSON.stringify` here. This is not Webpack environment, it's setting a specific value. So you don't want to add extra quotes around it.